projects/demo/src/modules/experimental/badge-alert/examples/1/index.html

```diff
@@ -0,0 +1,23 @@
+<h4>Desktop</h4>
+<p>
```

Comment:
You should add `tuiPlatform="web"` because in the future it will be determined automatically.